src/app.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const artistsRouter = require('./routes/artists');
+
+const app = express();
+
+app.use(express.json());
+
+app.use('/artists', artistsRouter);
```

Comment:
In the case of larger files with a lot of routes it would be good to separate the different parts with a clear line to make it easier to read at a glance. In this case though its fine to bunch them together since its so short

```
const app = express();
app.use(express.json());
app.use('/artists', artistsRouter);
```